frankenphp.c

```diff
@@ -430,12 +432,7 @@ static void frankenphp_request_shutdown() {
   php_request_shutdown((void *)0);
   frankenphp_free_request_context();
 
-  free(ctx);
-  SG(server_context) = ctx = NULL;
-
-#if defined(ZTS)
-  ts_free_thread();
-#endif
+  memset(local_ctx, 0, sizeof(frankenphp_server_context));
```

Comment:
Is it necessary, as we always call `memset` at startup? Ideally, calling it at shutdown on after the first creation should be better performance wise.

Reply:
I was wondering exactly the same thing :D I'll remove one.